target/linux/ath79/dts/ar7100_mikrotik_routerboard-4xx.dtsi

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &led_user;
+		led-failsafe = &led_user;
+		led-running = &led_user;
+	};
```

Comment:
Cosmetic, but I'd prefer to have the aliases first, i.e. before the chosen node.

Reply:
Swapped them over